

pandas.errors.CategoricalConversionWarning#


exception pandas.errors.CategoricalConversionWarning[source]#
Warning is raised when reading a partial labeled Stata file using a iterator.
Examples
>>> from pandas.io.stata import StataReader
>>> with StataReader('dta_file', chunksize=2) as reader: 
...   for i, block in enumerate(reader):
...      print(i, block)
... # CategoricalConversionWarning: One or more series with value labels...




